mypy/message_registry.py

```diff
@@ -112,7 +112,10 @@ def with_additional_msg(self, info: str) -> ErrorMessage:
     "This usage of this covariant type variable is unsafe as an input parameter.",
     codes.UNSAFE_VARIANCE,
 )
-UNSAFE_VARIANCE_NOTE = "If this is intentional and you know what you are doing, you can ignore this line with 'unsafe-variance'"
+UNSAFE_VARIANCE_NOTE = ErrorMessage(
+    "If you are using the value in a 'variance safe' way, this error could be ignored",
```

Comment:
can this maybe link to some documentation or something explaining what "variance safe" actually means

Reply:
What does it means?

Reply:
I think something like "not storing the value". but that's a little confisung for conta outputs, which should say "don't return a stored value".

Reply:
- How does that look? I think when we #260 we can have a section there.